Reviewing and fixing tutorials/gcc-use.md (#78)
canonical/ubuntu-for-developers-docs: docs/tutorials/gcc-use.md

    <span style="font-weight:bold;">hello.c:</span> In function ‘<span style="font-weight:bold;">main</span>’:
    <span style="font-weight:bold;">hello.c:5:57:</span> <span style="font-weight:bold;color:purple;">warning: </span>format ‘<span style="font-weight:bold;">%d</span>’ expects a matching ‘<span style="font-weight:bold;">int</span>’ argument [<span style="font-weight:bold;color:purple;">-Wformat=</span>]
        5 |     printf(&quot;Hello, world!\nThe answer to everything is <span style="font-weight:bold;color:purple;">%d</span>.\n&quot;);
          |                                                        <span style="font-weight:bold;color:purple;">~^</span>
          |                                                         <span style="font-weight:bold;color:purple;">|</span>
          |                                                         <span style="font-weight:bold;color:purple;">int</span></pre></div></div>
    :::

>     {command}`gcc` displays a warning, but still compiles the source code.

Reviewer: Removing the subject removes the subordinate clause.
```suggestion
    {command}`gcc` displays a warning but still compiles the source code.
```

    The answer to everything is -1350680728.
    ```

## Using the Make build system

> Compiling manually by invoking {command}`gcc` directly can help understand the build process, or for simple projects like 'Hello, world!'. However, a build system can be used to simplify and automate the process.

Reviewer: - The first sentence loses meaning with the modification.
- I don't agree with the modification of the second sentence. "Can be used" is vague, non-authoritative filler. How about this:
```suggestion
Compiling manually by invoking {command}`gcc` directly can help understand the build process. It is also adequate for simple projects like 'Hello, world!'. A build system simplifies and automates the process.
```
